### DISCUSSION TITLE
to_networkx is slow

### DISCUSSION THREAD

**Discussion by wubo2180** [pyg-team/pytorch_geometric#4270] [Q&A]
Hi,
In my code, I need to compute the centrality of graph nodes. However, this requires Networkx. Due to the large data set, there are about hundreds of thousands of graphs. So when using to_ to_networkx , the speed is very slow. Is there any way to speed up the calculation. Thanks!

**Comment by rusty1s** [comment]
I don't think so TBH. `networkx` is slow/does not scale by design since it makes use of Python lists. You can think about converting to other graph analytic libraries that support your use case better (for example, `cugraph`).
